AsyncDisplayKit/Private/ASInternalHelpers.h

```diff
@@ -33,6 +33,26 @@ BOOL ASRunningOnOS7();
 ASDISPLAYNODE_EXTERN_C_END
 
 /**
+ @summary Conditionally performs UIView geometry changes in the given block without animation and call completion block afterwards.
+ 
+ Used primarily to circumvent UITableView forcing insertion animations when explicitly told not to via
+ `UITableViewRowAnimationNone`. More info: https://github.com/facebook/AsyncDisplayKit/pull/445
+ 
+ @param withoutAnimation Set to `YES` to perform given block without animation
+ @param block Perform UIView geometry changes within the passed block
+ @param completion Call completion block if UIView geometry changes within the passed block did complete
+ */
+ASDISPLAYNODE_INLINE void ASPerformBlockWithoutAnimationCompletion(BOOL withoutAnimation, void (^block)(), void (^completion)()) {
```

Comment:
The UIKit version should just be a pure convenience for the Core Animation method. As we don't need the completion block function anymore I reverted the code back from the Core Animation to the UIKit one to prevent any regressions.

I will look into adding a protocol for need to check for shared behavior around check for batch fetching on every edited operations or similar functionality.